ci/recipe/meta.yaml

```diff
@@ -43,7 +43,7 @@ requirements:
     - psycopg2
     - pyarrow >=0.15
     - pydata-google-auth
-    - pymapd =0.23.0  # [py<38]
+    - pymapd 0.24  # [py<38]
```

Comment:
so you actually need to fix this version? this caused havoc the last time. I would rather wait until the smoke clears to touch this at all.

Reply:
@datapythonista thanks! yes, that would be great if it would be available for the next ibis release. thanks!